NEWS.md

```diff
@@ -1,5 +1,7 @@
 # lifecycle (development version)
 
+* You can now deprecate an argument with `foo(arg)`, not `foo(arg =)` (#78).
```

Comment:
```suggestion
* You can now deprecate an argument with `foo(arg)` instead of `foo(arg =)` (#78).
  This syntax is similar in spirit to the formal arguments of function definitions.
```